ci/terraform/oidc/api-gateway-frontend.tf

```diff
@@ -255,7 +255,7 @@ resource "aws_wafv2_web_acl" "wafregional_web_acl_frontend_api" {
     name     = "${var.environment}-frontend-waf-rate-based-rule"
     statement {
       rate_based_statement {
-        limit              = var.environment == "staging" ? 10000000 : 7200
+        limit              = var.environment == "staging" ? 10000000 : 100000
```

Comment:
I agree we can have a separate change for this , we already have lifecycle change ignore change in place for this resource  , this will be not implemented , new PR with agreed waf limit no

Reply:
yeah, this was definitely an oops - i'll get this removed.